app/models/vendor.rb

```diff
@@ -1,2 +1,13 @@
 class Vendor < ActiveRecord::Base
+  belongs_to :market
+  has_many :sales
+  has_many :products
+
+  def monthly_sales
+    @time_month = Time.now.month
```

Comment:
Are these instance variables used anywhere else? If not, they should be local variables instead